src/Client/V2/Access/DatabaseAccess.cs

```diff
@@ -750,7 +750,7 @@ public virtual Task<AsyncQueryData> ReadQueryAsync( string databaseId, string qu
         /// The optional call context to include.
         /// </param>
         public virtual async Task<AsyncQueryData> ReadQueryWhenReadyAsync( string databaseId, string queryId, int start, int end,
-            TimeSpan initialDelay, float backoffFactor = 1.25f, CancellationToken cancel = default, CallContext context = null )
+            TimeSpan initialDelay, float backoffFactor = 1.25f, System.Threading.CancellationToken cancel = default, CallContext context = null )
```

Comment:
I don't think you need this change. There is already a using for System.Threading.

Reply:
I ended up removing that using since it was only needed for the `CancellationToken`, thought it would be better like that but let me know if you think otherwise 🤭